test/harvest/test_merge_pubs.py

```diff
@@ -93,6 +97,7 @@ def openalex_pubs_csv(tmp_path):
                 "A Publication",
                 "article",
                 "https://doi.org/10.0000/cccc",
+                '{is_oa: true, landing_page_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1398957", pdf_url: null, source: { id: "https://openalex.org/S2764455111", display_name: "PubMed Central", issn_l: null, issn: null, host_organization: "https://openalex.org/I1299303238", type: "repository" }, license: null, version: "publishedVersion"}',
```

Comment:
THe value for `best_oa_location` is a JSON object.

Reply:
Will this end up as a string in our Parquet file? Maybe in a future step we could extract the fields that look useful into columns.

Reply:
Yes, it will be a python string representation for a dictionary unfortunately, which we will need to eval.

If we switched to harvesting as JSONL then Parquet could store it natively which would be easier to work with I think.